imports.go

```diff
@@ -2,7 +2,10 @@ package cayley
 
 import (
 	"github.com/google/cayley/graph"
+	_ "github.com/google/cayley/graph/bolt"
```

Comment:
I'd rather let the user choose the imports for non-memory graphs for their use case. Beyond that, the function below looks fine.